lib/isInteger.js

```diff
@@ -3,7 +3,18 @@
 // @returns {Boolean}
 
 function isInteger(param) {
-  return;
+  if(typeof param == "number"){
+    if((param % 1) === 0)
```

Comment:
Also- correct me if I'm wrong (it's late for me), but aren't negative numbers integers? This check would return false if `param === -123`.

Reply:
Maybe add that to the test suite!

Reply:
never mind.. I'm an idiot. `-0 === 0` Goodnight javascript!